tests/bench_sizeof_coretypes/README.md

```diff
@@ -0,0 +1,6 @@
+# Measure Size of Core Types
+
+This application simply list the size of all types defined by the RIOT core.
```

Comment:
Did a grep for all typedefs in core/include and found `schedstat`, couple of native types, byte ordering types aren't covered. Would it be worth changing this statement to say "except some types we don't care about" or something like that? For the sake of correctness?

Reply:
We could do that. Will fix.